### DISCUSSION TITLE
What about the re-pack

### DISCUSSION THREAD

**Discussion by izark1** [extremecoders-re/pyinstxtractor#100] [General]
This is a great script used to extract python executable. I used it to extract one package I have, did some modification on one file. Now, the question comes, how to repack all the content after the modification into one file?

**Comment by extremecoders-re** [comment]
Developing a repacker has been on the project timeline. Infact there is a prototype already built but that is specific to Python 3.7 and hence not made public. Meanwhile you can try out this tool: https://github.com/in1nit1t/pyipx

**Reply by extremecoders-re** [reply]
Just made it open-source anyway. The code hasn't been touched in years. Expect bugs.

https://github.com/pyinstxtractor/pyinstaller-repacker
